evennia/utils/evmenu.py

```diff
@@ -64,10 +64,63 @@ def another_node(caller, input_string, **kwargs):
 The return values must be given in the above order, but each can be
 returned as None as well. If the options are returned as None, the
 menu is immediately exited and the default "look" command is called.
+    text (str, tuple, dict or None): Text shown at this node. If a dict
+        it can also contain various formatting instructions. In such a
+        case the following keys are already supported:
+
+            - 'header' (dict): Dictionary of header data for the node
+            - 'text' (dict): Dictionary of text data for the node
+            - 'options' (dict): Dictionary of options data for the node
+            - 'footer' (dict): Dictionary of footer data for the node
+            - 'help' (dict): Dictionary of help data for the node
+
+            Each of these keys support their own dictionary with the
+            following keys:
+
+                - 'contents' (string or list): Required. String data for
+                    all keys other than 'options'. List in the case of
+                    options.
+                - 'formatter' (string): Optional. Designator of the
```

Comment:
This is now the name of the formatting function on the EvMenu class; this should be explained somewhere. It also makes sense to support the standard format here - `(formattername, {kwargs})`, where formatter is either a string (in this case, a method on the EvMenu class). 
(or an actual callable I presume, something like  `formattername(caller, evmenuinstance, **kwargs)`?)